mkdocs.yml

```
@@ -94,6 +96,19 @@ theme:
     # https://squidfunk.github.io/mkdocs-material/setup/setting-up-the-header/#mark-as-read
     - announce.dismiss
 
+    # Enable copy-to-clipboard button
+    # https://squidfunk.github.io/mkdocs-material/reference/code-blocks/#code-copy-button
+    - content.code.copy
+
+    # Enable view/edit source button.
+    # https://squidfunk.github.io/mkdocs-material/setup/adding-a-git-repository/#code-actions
+    - content.action.edit
+    - content.action.view
```

```suggestion
    # Enable edit source button.
    # https://squidfunk.github.io/mkdocs-material/setup/adding-a-git-repository/#code-actions
    - content.action.edit
```

There's a new option to show a "view source" button, but do we actually need it?

- Two buttons next to each other makes it easy to click on the "wrong" button.
- The buttons are small already.
- There's only a small space between the buttons.

I think most people want to edit the source file directly, with one button that's easy, just select the pencil button, and get editing. With two buttons you need to hover over them or know what the icon means to select the option you want.

This suggestion makes it so we only have the "edit page" button.